sources/commands/Cleanup.ts

```diff
@@ -0,0 +1,90 @@
+import {Command}          from 'clipanion';
+import fs                 from 'fs';
+import path               from 'path';
+import semver             from 'semver';
+
+import * as debugUtils    from '../debugUtils';
+import {getInstallFolder} from '../folderUtils';
+import type {Context}     from '../main';
+import type {NodeError}   from '../nodeUtils';
+import {parseSpec}        from '../specUtils';
+
+export class CleanupCommand extends Command<Context> {
+  static paths = [
+    [`cleanup`],
```

Comment:
What do you think of `corepack cache clean`, to mirror the same command from package managers?

```suggestion
    [`cache`, `clean`],
```